File: wrappers/dotnet/indy-sdk-dotnet/IOException.cs
namespace Hyperledger.Indy
{
    /// <summary>
    /// Exception indicating that an IO error occurred.
    /// </summary>
    public class IOException : IndyException
    {
        const string message = "An IO error occurred.";

        /// <summary>
        /// Initializes a new IOException.
        /// </summary>
        internal IOException() : base(message, (int)ErrorCode.CommonIOError)
        {

        }
    }

}

File: wrappers/android/src/main/java/org/hyperledger/indy/sdk/IOException.java
package org.hyperledger.indy.sdk;

/**
 * Exception thrown when the SDK experienced an IO error.
 */
public class IOException extends IndyException
{
	private static final long serialVersionUID = -1581785238453075780L;
	private final static String message = "An IO error occurred.";

	/**
	 * Initializes a new IOException.
	 */
	public IOException()
	{
		super(message, ErrorCode.CommonIOError.value());
	}
}